### PR TITLE
Update protobuf and grpc to fix CVE-2022-3171

### PR DESCRIPTION
**Pull request by Torch3333**
This fixes CVE-2022-3171
I verified theses changes are backward compatible.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -32,8 +32,8 @@ subprojects {
         postgresqlDriverVersion = '42.4.1'
         oracleDriverVersion = '19.8.0.0'
         sqlserverDriverVersion = '8.4.1.jre8'
-        grpcVersion = '1.46.0'
-        protobufVersion = '3.20.1'
+        grpcVersion = '1.46.1'
+        protobufVersion = '3.20.3'
```
Comment (Torch3333) [inline build.gradle:36]: The grpcVersion 1.46.0 was not compatible with the code generated by protobuf version 3.20.3, so it had to be updated to 1.46.1

**Review by brfrn169 — APPROVED** [review]
LGTM! Thank you! 

Since this change upgrades the libs between patch versions, it should be backward compatible.

**Review by komamitsu — APPROVED** [review]
LGTM! 👍

**Review by feeblefakie — APPROVED** [review]
LGTM! Thank you!

**Pull request merged**